projects/ngx-feature-viewer/ng-package.json

```diff
@@ -19,7 +19,9 @@
     "npx",
     "sass",
     "typed-css-modules",
-    "underscore"
+    "underscore",
+    "@types/lodash-es",
+    "lodash-es"
```

Comment:
Why do you need lodash when you have underscore?

Reply:
Because it does not have 'cloneDeep'